src/Test.Utilities/CodeMetricsTestsBase.cs

```diff
@@ -68,7 +68,14 @@ private void Verify(string[] sources, string expectedMetricsText, bool expectDia
             var project = CreateProject(sources, language);
             var compilation = project.GetCompilationAsync(CancellationToken.None).Result;
             var diagnostics = compilation.GetDiagnostics().Where(d => d.Severity == DiagnosticSeverity.Warning || d.Severity == DiagnosticSeverity.Error);
-            Assert.Equal(expectDiagnostics, diagnostics.Any());
```

Comment:
Why was this change required?

Reply:
It wasn't required to make the tests pass or anything, but I found it useful when a test that was not expecting diagnostics to appear caused some to appear, to have a list of those diagnostics in the output instead of just "expected false but found true."

If you don't like it, I can easily back it out.